app/graphql/types/movie_type.rb

```diff
@@ -26,6 +26,10 @@ class MovieType < Base::Object
           null: true,
           description: I18n.t("#{I18N_PATH}.fields.poster")
 
+    field :removed_movie_id, ID, null: false, description: I18n.t("#{I18N_PATH}.fields.removed_movie_id")
```

Comment:
please move this field to the separate type

Reply:
Moved